decidim-participatory_processes/app/models/decidim/participatory_process_private_user.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This class gives a given User access to a given private Process
+  class ParticipatoryProcessPrivateUser < ApplicationRecord
```

Comment:
Again, this look really the same as assemblies. Why not ParticipatorySpacePrivateUser? This way it can be reused for other kind of spaces.